requirements.txt

```diff
@@ -79,7 +79,7 @@ flask-talisman==0.8.1
     # via -r requirements.in
 flask-wtf==1.0.0
     # via -r requirements.in
-funding-service-design-utils @ https://github.com/communitiesuk/funding-service-design-utils/archive/refs/tags/0.0.19.tar.gz
+funding-service-design-utils @ git+https://github.com/communitiesuk/funding-service-design-utils@bau-notify-fields
```

Comment:
Can we change this to a released version?

Reply:
Will do once I've merged this one https://github.com/communitiesuk/funding-service-design-utils/pull/23